fix: New combat sounds (#958)
Zenoo/labrute: README.md

Should have a corresponding documentation.

> ## Shape all sound in one sound with sprites using audiosprite

Reviewer: ```suggestion
## Generate the sound spritesheet after editing a sound using audiosprite
```

> Command line in the sound folder : `audiosprite $(find . -type f -name '_.mp3' -o -name '_.wav') -o mybruteaudio -u .`

Reviewer: ```suggestion
- Run the following command in the sound folder : `audiosprite $(find . -type f -name '_.mp3' -o -name '_.wav') -o mybruteaudio -u .`
```

Please rename mybryteaudio to sfx here too (and the file created too).
Also, it would be better to add a script in package.json, and call it at the root (`yarn sfx:generate`) for example, which would execute the above command in the correct folder.

> 
> audiosprite will check every folder of the location that has mp3 or wav extension and fuse them in one sound named "sfx"
> There will be 4 outputs : mp3, gg, m4a and ac3.

Reviewer: ```suggestion
- audiosprite will gather every mp3 or wav sound in the sfx folder and fuse them in one sound named "sfx"
```

**You can just keep one of them (mp3 os the more common)**
**You have to adapt the resources values ( in this case add '/sfx/')**

> **You have to move the json file in the src folder because it cannot be import from the public folder**

Reviewer: These 4 last lines should be handled automatically by the `yarn sfx:generate` command without user intervention.
